src/Express.res

```diff
@@ -0,0 +1,624 @@
+    |> Js.Dict.fromArray
+  }
+  let cookie = (
+    ~name,
+    ~maxAge=?,
+    ~expiresGMT=?,
+    ~httpOnly=?,
+    ~secure=?,
+    ~signed=?,
+    ~path=?,
+    ~sameSite: option<sameSite>=?,
+    ~domain=?,
+    value,
+    response,
+  ) => {
+    cookie_(
+      name,
+      value,
+      {
+        "maxAge": maxAge |> Js.Nullable.fromOption,
+        "expires": expiresGMT |> Js.Nullable.fromOption,
+        "path": path |> Js.Nullable.fromOption,
+        "httpOnly": httpOnly |> Js.Nullable.fromOption,
+        "secure": secure |> Js.Nullable.fromOption,
+        "sameSite": sameSite |> Js.Option.map((. x) => sameSiteToJs(x)) |> Js.Nullable.fromOption,
+        "signed": signed |> Js.Nullable.fromOption,
+        "domain": domain |> Js.Nullable.fromOption,
+      } |> filterKeys,
+      response,
+    )
+    response
+  }
+  let clearCookie = (
+    ~name,
+    ~httpOnly=?,
+    ~secure=?,
+    ~signed=?,
+    ~path="/",
+    ~sameSite: option<sameSite>=?,
+    response,
+  ) => {
+    clearCookie_(
+      name,
+      {
+        "maxAge": Js.Nullable.undefined,
+        "expires": Js.Nullable.undefined,
+        "path": path,
+        "httpOnly": httpOnly |> Js.Nullable.fromOption,
+        "secure": secure |> Js.Nullable.fromOption,
+        "sameSite": sameSite |> Js.Option.map((. x) => sameSiteToJs(x)) |> Js.Nullable.fromOption,
+        "signed": signed |> Js.Nullable.fromOption,
+      } |> filterKeys,
+      response,
+    )
+    response
+  }
+  @bs.send.pipe(: t) external sendFile: (string, 'a) => complete = "sendFile"
+  @bs.send.pipe(: t) external sendString: string => complete = "send"
+  @bs.send.pipe(: t) external sendJson: Js.Json.t => complete = "json"
+  @bs.send.pipe(: t) external sendBuffer: Node.Buffer.t => complete = "send"
+  @bs.send.pipe(: t) external sendArray: array<'a> => complete = "send"
+  @bs.send.pipe(: t) external sendRawStatus: int => complete = "sendStatus"
+  let sendStatus = statusCode => sendRawStatus(StatusCode.toInt(statusCode))
+  @bs.send.pipe(: t) external rawStatus: int => t = "status"
+  let status = statusCode => rawStatus(StatusCode.toInt(statusCode))
+  @bs.send.pipe(: t) @ocaml.deprecated("Use sendJson instead`")
+  external json: Js.Json.t => complete = "json"
+  @bs.send.pipe(: t)
+  external redirectCode: (int, string) => complete = "redirect"
+  @bs.send.pipe(: t) external redirect: string => complete = "redirect"
+  @bs.send.pipe(: t) external setHeader: (string, string) => t = "set"
+  @bs.send.pipe(: t) external setType: string => t = "type"
+  @bs.send.pipe(: t) external setLinks: Js.Dict.t<string> => t = "links"
+  @bs.send.pipe(: t) external end_: complete = "end"
+  @bs.send.pipe(: t) external render: (string, Js.Dict.t<string>, 'a) => complete = "render"
+}
+
+module Next: {
+  type content
+  type t = (Js.undefined<content>, Response.t) => complete
+  let middleware: Js.undefined<content>
+
+  @ocaml.doc("value to use as [next] callback argument to invoke the next middleware")
```

Comment:
Might have been like this before, but the doc comments don't seem to line up with the functionality. For the implementation `*.res` file, actually don't think the comments are necessary anyway

Reply:
Perfect. Thanks a lot!